ui/app/src/styles/card.ts

```diff
@@ -0,0 +1,29 @@
+/*!
```

Comment:
Since you're moving the "title card" into GlobalNav, you won't need this file either.